go.mod

```diff
@@ -48,6 +49,7 @@ require (
 	github.com/muesli/reflow v0.3.0
 	github.com/nqd/flat v0.2.0
 	github.com/otiai10/copy v1.11.0
+	github.com/ozgur-yalcin/google-analytics v0.0.0-20230220005025-ed55dad8f7a1
```

Comment:
Should we really add a dependency of a project with 13 stars? https://github.com/ozgur-yalcin/google-analytics
I know it is a vanity metric, but it has no tag, and no dependent: https://pkg.go.dev/github.com/ozgur-yalcin/google-analytics

Reply:
Is a simple logic, and we don't need to reinvent the wheel. Do you have another solution to share?

Reply:
Or do you think we should implement the API by ourselves?

Reply:
My issue is mostly about long-term maintenance and the fact that if the packages get deleted, it will just break ignite.
Personally, I think we should copy/paste the code in here and credit the repo as header. This way there is no risk anymore.